docs/modules/ROOT/pages/authentication-and-authorization/configuration.adoc

```diff
@@ -0,0 +1,125 @@
+= Configuration
+
+== Instantiation
+
+The Neo4j GraphQL Library can accept JSON Web Tokens via two mechanisms:
+
+* Encoded JWTs in the `token` field of the request context.
+* Decoded JWTs in the `jwt` field of the request context.
+
+If using the former, the library will need to be configured with a key to decode and verify the token.
+
+The following code block demonstrates using Apollo Server, extracting the `Authorization` header from the request and putting it into the appropriate context field:
+
+[source, typescript, indent=0]
+----
+const server = new ApolloServer({
+    schema, // schema from Neo4jGraphQL.getSchema()
+});
+
+const { url } = await startStandaloneServer(server, {
+    listen: { port: 4000 },
+    context: async ({ req }) => ({
+        token: req.headers.Authorization,
+    }),
+});
+----
+
+Optionally, if a custom decoding mechanism is required, that same header can be decoded and the resulting JWT payload put into the `jwt` field of the context.
+
+=== Symmetric secret
+
+To configure the library with a symmetric secret, "secret", the following instantiation is required:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    features: {
+        authorization: {
+            key: "secret",
+        },
+    },
+});
+----
+
+=== JWKS endpoint
+
+To configure the library to verify tokens against a JWKS endpoint, "https://www.myapplication.com/.well-known/jwks.json", the following instantiation is required:
```

Comment:
```suggestion
To configure the library to verify tokens against a https://www.myapplication.com/.well-known/jwks.json[JWKS endpoint], the following instantiation is required:
```

Reply:
This one shouldn't actually be a link, it's just giving the address which will be used in the example below it.